src/main/java/org/embl/mobie/viewer/command/CropSourcesCommand.java

```diff
@@ -64,6 +62,8 @@ public void run()
 				final SourceAndConverter cropSource = cropSource( maskInterval, maskTransform, sourceAndConverter );
 
 				// TODO: can we create a view for this?
+				final MoBIE moBIE = MoBIE.getInstance( bdvHandle );
```

Comment:
@K-Meech Do you have experience here and could help? The point is to somehow create and register a new `view` for the `cropSource` (actually this is a `MaskedSource`, need to change names later, maybe).

Reply:
I think you would need to create the relevant ```SourceTransformer``` by getting the min/max of the crop etc from the Source. Then somehow add it to the the ```currentSourceTransformers``` in the ```ViewManager```. Then, when you save the current settings as a view, it will be made into a view when this function is called: https://github.com/mobie/mobie-viewer-fiji/blob/develop/src/main/java/org/embl/mobie/viewer/view/ViewManager.java#L208